en-US/firefox/campaign.lang

```diff
@@ -1,28 +1,7 @@
-## download_thanks_newsletter_092018 ##
-## firefox_new_platform_link_text ##
-## firefox_new_try_again_link_020818 ##
-## firefox_quantum_new_headline ##
-## firstrun_email_only_form ##
 ## NOTE: Demo page at https://www-dev.allizom.org/firefox/campaign/
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/campaign/
 
 
-;Congrats! You’re using the latest version of Firefox.
-Congrats! You’re using the latest version of Firefox.
-
-
-;<a href="%(url)s">Update</a> your Firefox for the latest in speed and privacy.
-<a href="%(url)s">Update</a> your Firefox for the latest in speed and privacy.
-
-
-;You’re using a pre-release version of Firefox.
-You’re using a pre-release version of Firefox.
-
-
-;You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.
-You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.
-
-
 ## TAG: firefox_quantum_new_headline
```

Comment:
I see an error locally:

```
The following tags are bound to strings but not declared at the beginning of the file:

    firefox_quantum_new_headline
``` 

In other words: you have a tag associated to a string, but there's no tag declared at the beginning of the file. I'll clean it up.